.gitignore

```diff
@@ -1,3 +1,4 @@
 _site/
 .jekyll-metadata
 node_modules
+.idea/
```

Comment:
What's this?

Reply:
its to leave out intellij settings from the repo

Reply:
I would drop this commit since it's only for your local machine.